configs/katello_33.yaml

```diff
@@ -0,0 +1,178 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+  :3.2.3:
+    :redmine_release_id: 214
+:releases:
+  :3.3:
+    :redmine_release_id: 188
+:code_name: Baltic Porter
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.3
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :katello-selinux:
```

Comment:
sorry, i see what you mean now.  yes there were (from months ago).  I've updated that and started a new build in katello-packaging